src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -864,9 +864,9 @@ namespace Akka.Actor
         public Failures() { }
         public System.Collections.Generic.List<Akka.Actor.Failure> Entries { get; }
     }
-    public class FutureActorRef : Akka.Actor.MinimalActorRef
+    public class FutureActorRef<T> : Akka.Actor.MinimalActorRef
```

Comment:
Technically a breaking API change, but `FutureActorRef` isn't used as part of the public API. It's only used internally by `Ask<T>` (and maybe one other odd thing here or there)